app/src/main/java/swati4star/createpdf/fragment/HomeFragment.java

```diff
@@ -72,6 +75,9 @@
     private String mPath;
     private String mFilename;
     private String mPassword;
+    private View positiveAction;
```

Comment:
change `positiveAction` to `mPositiveAction`
`neutralAction` to `mNeutralAction`
`passwordInput` to `mPasswordInput`